workflow/kfdrc_production_controlfreec_wf.cwl

```diff
@@ -41,7 +41,7 @@ inputs:
   mate_copynumber_file_control: {type: 'File?', doc: "Normal cpn file from previous run. If used, will override bam use"}
   mate_copynumber_file_sample: {type: 'File?', doc: "Tumor cpn file from previous run. If used, will override bam use"}
   gem_mappability_file: {type: 'File?', doc: "GEM mappability file to make read count adjustments with"}
-  min_subclone_presence: {type: 'float?', doc: "Use if you want to detect sublones. Recommend 0.2 for WGS, 0.3 for WXS"}
+  min_subclone_presence: {type: 'int?', doc: "Use if you want to detect sublones. Recommend 30 for WGS, 30 for WXS"}
```

Comment:
Why is it 30 here but 20 in the subworkflow?

Reply:
Good catch, defo a typo!